documentation/content/awesome_kw.rst

```diff
@@ -0,0 +1,31 @@
+===================
+ Awesome Kworkflow
+===================
```

Comment:
Keep the `===...` under the title size.

Maybe rename the title to 'Awesome kworkflow related projects' or something similar.

Reply:
The term _Awesome Y_  is a well known term that describes a collection of _awesome_ tools for _Y_.

See: [awesome-rust](https://github.com/rust-unofficial/awesome-rust), [awesome-go](https://github.com/avelino/awesome-go), [awesome-nu](https://github.com/nushell/awesome-nu)